employee-rostering-frontend/src/store/contract/operations.ts

```diff
@@ -45,14 +44,15 @@ export const updateContract: ThunkCommandFactory<Contract, UpdateContractAction>
     const tenantId = contract.tenantId;
     return client.post<Contract>(`/tenant/${tenantId}/contract/update`, contract).then(updatedContract => {
       dispatch(actions.updateContract(updatedContract));
-      dispatch(employeeOperations.refreshEmployeeList());
```

Comment:
It is redundant now; since `EmployeeList` is now a `id -> DomainObjectView<Employee>` map, an `Employee` in the map no longer have the `contract` hard coded into the object. Rather, they hold an `id`, and you need to use an `employee` selector to get the `Employee` list with the Contract.

Reply:
Yes, of course! I forgot this is one of the reasons we're normalizing the state :slightly_smiling_face:.